src/Core/Components/List/FluentAutocomplete.razor.cs

```diff
@@ -513,6 +513,34 @@ async Task KeyDown_EnterAsync()
         }
     }
 
+    /// <summary />
+    protected override void OnParametersSet()
+    {
+        base.OnParametersSet();
+
+        if (MaximumSelectedOptions == 1
+            && SelectedOptions?.Count() > 0
```

Comment:
For better performances, you can use `SelectedOptions?.Any()` (and not `Count()`

Reply:
Done